settings.gradle

```diff
@@ -1,6 +1,5 @@
 pluginManagement {
     repositories {
-        jcenter()
```

Comment:
So the million dollar question, are there any dependencies on jcenter we have to worry about?

Reply:
nick already said:
> I did not replace this with another repoistory (e.g. mavenCentral()) since there seem to be no dependencies that relied on it.